sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -1290,12 +1380,18 @@ def replace(self, to_replace, value, limit, method, **kwargs):
 
   to_frame = frame_base._elementwise_method('to_frame', base=pd.Series)
 
+  @frame_base.with_docs_from(pd.Series)
   def unique(self, as_series=False):
+    """unique() is not supported by default because it produces a
+    non-deferred result, a numpy array. You may use the Beam-specific argument
```

Comment:
"non-deferred result, a numpy array. You may use the Beam-specific argument" -> "non-deferred result: a numpy array. You can use the Beam-specific argument"

Reply:
Done